examples/seismic/tti/tti_example.py

```diff
@@ -41,14 +35,14 @@ def tti_setup(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
 
 
 def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
-        time_order=2, space_order=4, nbpml=10, **kwargs):
+        time_order=2, space_order=4, nbpml=10, kernel='centered', **kwargs):
     autotune = kwargs.pop('autotune', False)
```

Comment:
you've undone ``autotune`` in the signature

Reply:
Not part of the PR so yes